config/core/test-configs.yaml

```diff
@@ -33,36 +33,36 @@ file_systems:
 
   debian_buster_ramdisk:
     type: debian
-    ramdisk: 'buster/20210503.0/{arch}/rootfs.cpio.gz'
+    ramdisk: 'buster/20210520.0/{arch}/rootfs.cpio.gz'
 
   debian_buster_nfs:
     type: debian
-    ramdisk: 'buster/20210503.0/{arch}/initrd.cpio.gz'
-    nfs: 'buster/20210503.0/{arch}/full.rootfs.tar.xz'
+    ramdisk: 'buster/20210520.0/{arch}/initrd.cpio.gz'
+    nfs: 'buster/20210520.0/{arch}/full.rootfs.tar.xz'
     root_type: nfs
 
   debian_buster-cros-ec_ramdisk:
     type: debian
-    ramdisk: 'buster-cros-ec/20210503.0/{arch}/rootfs.cpio.gz'
+    ramdisk: 'buster-cros-ec/20210520.0/{arch}/rootfs.cpio.gz'
 
   debian_buster-igt_ramdisk:
     type: debian
-    ramdisk: 'buster-igt/20210503.0/{arch}/rootfs.cpio.gz'
+    ramdisk: 'buster-igt/20210520.0/{arch}/rootfs.cpio.gz'
 
   debian_buster_kselftest:
     type: debian
-    ramdisk: 'buster-kselftest/20210514.0/{arch}/rootfs.cpio.gz'
-    nfs: 'buster-kselftest/20210514.0/{arch}/full.rootfs.tar.xz'
+    ramdisk: 'buster-kselftest/20210520.0/{arch}/rootfs.cpio.gz'
```

Comment:
@gctucker the problem is here, ramdisk can not be rootfs, it needs to initrd, right? ;)

Reply:
https://lava.collabora.co.uk/scheduler/job/3871673

Reply:
Yes, good point.  Please send a fix for that and I'll skip the revert PR #692.